src/test/java/uk/co/ramp/covid/simulation/integrationTests/LockdownTests.java

```diff
@@ -1,5 +1,6 @@
 package uk.co.ramp.covid.simulation.integrationTests;
 
+import org.junit.Ignore;
```

Comment:
(Just looking at the one commit,) in this file, you appear to have changed some whitespace and added an unused import statement.  I guess you committed this accidentally?